leaderelection/chaosduck/main.go

```diff
@@ -45,13 +46,15 @@ type components map[string]sets.String
 
 var (
 	disabledComponents kflag.StringSet
-	tributePeriod      time.Duration = 30 * time.Second
+	tributePeriod      time.Duration = 20 * time.Second
+	tributeFactor                    = 2.0
 )
 
 func init() {
 	// Note that we don't explicitly call flag.Parse() because ParseAndGetConfigOrDie below does this already.
 	flag.Var(&disabledComponents, "disable", "A repeatable flag to disable chaos for certain components.")
-	flag.DurationVar(&tributePeriod, "period", tributePeriod, "How frequently to terminate a leader pod per component.")
+	flag.DurationVar(&tributePeriod, "period", tributePeriod, "How frequently to terminate a leader pod per component (this is the base duration used with the jitter factor from -factor).")
+	flag.Float64Var(&tributeFactor, "factor", tributeFactor, "The jitter factor to apply to the period.")
```

Comment:
why not just `flag.Duration`, etc?

Reply:
AFAIK can't do that for for `flag.Var`, so we needed `init()` anyways, so I avoided the annoying pointers you usually get 😉